src/example-types/filters/tagsQuery.js

```diff
@@ -80,6 +80,50 @@ let filter = ({ tags, join, field, exact }) => ({
   },
 })
 
+let result = async (node, search, schema, { options, getProvider }) => {
+  let { tags, join, field, exact, _meta, maxToCount = 20 } = node
+  if (_.size(tags) > maxToCount) {
```

Comment:
This should probably live in the `validContext` method - checking it here would be if you want to let the first 20 through. If we're just bailing on the query, semantically this check is about determining if the node is valid.

Reply:
good point 👍

Reply:
done -- no need for a max IMO in the agg version